code/game/objects/items/plushes.dm

```diff
@@ -696,6 +721,84 @@
 	desc = "Hgrgrhrhg cute."
 	icon_state = "flushplush"
 
+/obj/item/choice_beacon/radial/plushie
+	name = "plushie delivery beacon"
+	desc = "Summon your new friend!"
+	icon_state = "gangtool-plushie"
+	var/static/list/plushie_list
+
+/obj/item/choice_beacon/radial/plushie/Initialize(mapload)
```

Comment:
you don't even need to stick this in /Initialize, can just put the = list(...) right next to the static declaration